.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +38,11 @@ Test case added to lib/integration.sh
 If no, create an issue with enhancement/testing label
 -->
 
+** Will this PR introduce any new dependencies?**:
+<!-- 
+e.g. new EC2/K8s API calls, dependency on specific kernel module/version or binary in container OS.
```

Comment:
Maybe we can add `IMDS` specifically here as well and make it explicit.

Reply:
make sense, updated it :D